src/FakeItEasy.ruleset

```diff
@@ -5,6 +5,7 @@
     <Rule Id="CA1020" Action="None" />
     <Rule Id="CA1200" Action="None" />
     <Rule Id="CA1305" Action="None" />
+    <Rule Id="CA2237" Action="None" /> <!-- Add [Serializable] to [...] as this type implements ISerializable -->

```

Comment:
Opinionated suppression: I don't see much point in marking our exceptions as serializable. Open to change.

Reply:
I'm in complete agreement